processor/src/tasks/processGerbers/index.ts

```diff
@@ -223,6 +223,8 @@ export default async function processGerbers(
       job.updateProgress({ status: 'failed', file, error, outputDir })
     }
   }
+
+  await Promise.all(filePaths.map(fp => fs.rm(fp, { force: true })))
```

Comment:
`top-meta.png` isn't in `filePaths`
![image](https://user-images.githubusercontent.com/37152329/204692080-b4fdf42d-bd01-4d46-ae39-6352c7768eab.png)

Reply:
Fixed with 378da7b7c2b8195